<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="6" total="15" passed="9">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="4530" started-at="2016-11-18T19:54:10Z" finished-at="2016-11-18T19:54:15Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="4530" started-at="2016-11-18T19:54:10Z" finished-at="2016-11-18T19:54:15Z">
      <class name="com.l1.interop.SPSPClientProxyFunctionalTest">
        <test-method status="PASS" signature="setup()[pri:0, instance:com.l1.interop.SPSPClientProxyFunctionalTest@5606c0b]" name="setup" is-config="true" duration-ms="1052" started-at="2016-11-18T12:54:10Z" finished-at="2016-11-18T12:54:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="beforeClass()[pri:0, instance:com.l1.interop.SPSPClientProxyFunctionalTest@5606c0b]" name="beforeClass" is-config="true" duration-ms="2" started-at="2016-11-18T12:54:11Z" finished-at="2016-11-18T12:54:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method status="FAIL" signature="invoice_GetInvoiceDetails_ForInvalidInvoice_ShouldReceive404Response(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.SPSPClientProxyFunctionalTest@5606c0b]" name="invoice_GetInvoiceDetails_ForInvalidInvoice_ShouldReceive404Response" duration-ms="2095" started-at="2016-11-18T12:54:11Z" data-provider="invoice_GET_negative" finished-at="2016-11-18T12:54:13Z">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
The data provider is trying to pass 1 parameters but the method com.l1.interop.SPSPClientProxyFunctionalTest#invoice_GetInvoiceDetails_ForInvalidInvoice_ShouldReceive404Response takes 9 and TestNG is unable in inject a suitable object]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
The data provider is trying to pass 1 parameters but the method com.l1.interop.SPSPClientProxyFunctionalTest#invoice_GetInvoiceDetails_ForInvalidInvoice_ShouldReceive404Response takes 9 and TestNG is unable in inject a suitable object
	at org.testng.internal.Invoker.injectParameters(Invoker.java:1361)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1225)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
Caused by: java.lang.ArrayIndexOutOfBoundsException: 1
	at org.testng.internal.Invoker.injectParameters(Invoker.java:1359)
	... 17 more
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invoice_GetInvoiceDetails_ForInvalidInvoice_ShouldReceive404Response -->
        <test-method status="FAIL" signature="invoice_GetInvoiceDetails_ForInvalidInvoice_ShouldReceive404Response(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.SPSPClientProxyFunctionalTest@5606c0b]" name="invoice_GetInvoiceDetails_ForInvalidInvoice_ShouldReceive404Response" duration-ms="2044" started-at="2016-11-18T12:54:11Z" data-provider="invoice_GET_negative" finished-at="2016-11-18T12:54:13Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Bob]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[http://localhost:8082/spsp/client/v1/invoice?invoiceUrl=http://brian.com]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[dfsp2.bob.dylan.account]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Bob Dylan]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[USD]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[$]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[999.99]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[unpaid]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[https://www.example.com/gp/your-account/order-details?ie=UTF8&orderID=111-7777777-1111111]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[1 expectation failed.
Expected status code <404> doesn't match actual status code <200>.
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 1 expectation failed.
Expected status code <404> doesn't match actual status code <200>.

	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:80)
	at org.codehaus.groovy.reflection.CachedConstructor.doConstructorInvoke(CachedConstructor.java:74)
	at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrap.callConstructor(ConstructorSite.java:84)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:60)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:235)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:247)
	at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:470)
	at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:48)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:113)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)
	at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:632)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:210)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:59)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:52)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:154)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:166)
	at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:121)
	at io.restassured.specification.ResponseSpecification$statusCode$2.callCurrent(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:52)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:154)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:166)
	at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:129)
	at io.restassured.internal.ValidatableResponseOptionsImpl.statusCode(ValidatableResponseOptionsImpl.java:117)
	at com.l1.interop.SPSPClientProxyFunctionalTest.invoice_GetInvoiceDetails_ForInvalidInvoice_ShouldReceive404Response(SPSPClientProxyFunctionalTest.java:750)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invoice_GetInvoiceDetails_ForInvalidInvoice_ShouldReceive404Response -->
        <test-method status="FAIL" signature="invoice_GetInvoiceDetails_ForValidInvoice_ShouldReceiveInvoiceDetailValidResponse(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.SPSPClientProxyFunctionalTest@5606c0b]" name="invoice_GetInvoiceDetails_ForValidInvoice_ShouldReceiveInvoiceDetailValidResponse" duration-ms="793" started-at="2016-11-18T12:54:13Z" data-provider="invoice_GET_positive" finished-at="2016-11-18T12:54:14Z">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
The data provider is trying to pass 1 parameters but the method com.l1.interop.SPSPClientProxyFunctionalTest#invoice_GetInvoiceDetails_ForValidInvoice_ShouldReceiveInvoiceDetailValidResponse takes 9 and TestNG is unable in inject a suitable object]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
The data provider is trying to pass 1 parameters but the method com.l1.interop.SPSPClientProxyFunctionalTest#invoice_GetInvoiceDetails_ForValidInvoice_ShouldReceiveInvoiceDetailValidResponse takes 9 and TestNG is unable in inject a suitable object
	at org.testng.internal.Invoker.injectParameters(Invoker.java:1361)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1225)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
Caused by: java.lang.ArrayIndexOutOfBoundsException: 1
	at org.testng.internal.Invoker.injectParameters(Invoker.java:1359)
	... 17 more
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invoice_GetInvoiceDetails_ForValidInvoice_ShouldReceiveInvoiceDetailValidResponse -->
        <test-method status="PASS" signature="invoice_GetInvoiceDetails_ForValidInvoice_ShouldReceiveInvoiceDetailValidResponse(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.SPSPClientProxyFunctionalTest@5606c0b]" name="invoice_GetInvoiceDetails_ForValidInvoice_ShouldReceiveInvoiceDetailValidResponse" duration-ms="683" started-at="2016-11-18T12:54:13Z" data-provider="invoice_GET_positive" finished-at="2016-11-18T12:54:14Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Bob]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[http://localhost:8081/spsp/client/v1/invoice?invoiceUrl=http://brian.com]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[dfsp2.bob.dylan.account]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Bob Dylan]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[USD]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[$]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[10.4]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[unpaid]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[https://www.example.com/gp/your-account/order-details?ie=UTF8&orderID=111-7777777-1111111]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invoice_GetInvoiceDetails_ForValidInvoice_ShouldReceiveInvoiceDetailValidResponse -->
        <test-method status="PASS" signature="invoice_GetInvoiceDetails_ForValidInvoice_ShouldReceiveInvoiceDetailValidResponse(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.SPSPClientProxyFunctionalTest@5606c0b]" name="invoice_GetInvoiceDetails_ForValidInvoice_ShouldReceiveInvoiceDetailValidResponse" duration-ms="106" started-at="2016-11-18T12:54:14Z" data-provider="invoice_GET_positive" finished-at="2016-11-18T12:54:14Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Sam]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[http://localhost:8081/spsp/client/v1/invoice?invoiceUrl=http://brian.com]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[dfsp2.sam.jones.account]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Bob Dylan]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[USD]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[$]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[10.4]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[unpaid]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[https://www.example.com/gp/your-account/order-details?ie=UTF8&orderID=111-7777777-1111111]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invoice_GetInvoiceDetails_ForValidInvoice_ShouldReceiveInvoiceDetailValidResponse -->
        <test-method status="FAIL" signature="invoice_Get_Ensure404WithInvalidURL_ShouldReceive404Response(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.SPSPClientProxyFunctionalTest@5606c0b]" name="invoice_Get_Ensure404WithInvalidURL_ShouldReceive404Response" duration-ms="39" started-at="2016-11-18T12:54:14Z" data-provider="invoice_GET_negative" finished-at="2016-11-18T12:54:14Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Bob]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[http://localhost:8082/spsp/client/v1/invoice?invoiceUrl=http://brian.com]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[dfsp2.bob.dylan.account]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Bob Dylan]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[USD]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[$]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[999.99]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[unpaid]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[https://www.example.com/gp/your-account/order-details?ie=UTF8&orderID=111-7777777-1111111]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[1 expectation failed.
Expected status code <404> doesn't match actual status code <200>.
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 1 expectation failed.
Expected status code <404> doesn't match actual status code <200>.

	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:80)
	at org.codehaus.groovy.reflection.CachedConstructor.doConstructorInvoke(CachedConstructor.java:74)
	at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrap.callConstructor(ConstructorSite.java:84)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:247)
	at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:470)
	at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
	at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:632)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:210)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:59)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:166)
	at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:121)
	at io.restassured.specification.ResponseSpecification$statusCode$2.callCurrent(Unknown Source)
	at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:129)
	at io.restassured.internal.ValidatableResponseOptionsImpl.statusCode(ValidatableResponseOptionsImpl.java:117)
	at com.l1.interop.SPSPClientProxyFunctionalTest.invoice_Get_Ensure404WithInvalidURL_ShouldReceive404Response(SPSPClientProxyFunctionalTest.java:814)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invoice_Get_Ensure404WithInvalidURL_ShouldReceive404Response -->
        <test-method status="FAIL" signature="invoice_Get_Ensure404WithInvalidURL_ShouldReceive404Response(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.SPSPClientProxyFunctionalTest@5606c0b]" name="invoice_Get_Ensure404WithInvalidURL_ShouldReceive404Response" duration-ms="41" started-at="2016-11-18T12:54:14Z" data-provider="invoice_GET_negative" finished-at="2016-11-18T12:54:14Z">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
The data provider is trying to pass 1 parameters but the method com.l1.interop.SPSPClientProxyFunctionalTest#invoice_Get_Ensure404WithInvalidURL_ShouldReceive404Response takes 9 and TestNG is unable in inject a suitable object]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
The data provider is trying to pass 1 parameters but the method com.l1.interop.SPSPClientProxyFunctionalTest#invoice_Get_Ensure404WithInvalidURL_ShouldReceive404Response takes 9 and TestNG is unable in inject a suitable object
	at org.testng.internal.Invoker.injectParameters(Invoker.java:1361)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1225)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
Caused by: java.lang.ArrayIndexOutOfBoundsException: 1
	at org.testng.internal.Invoker.injectParameters(Invoker.java:1359)
	... 17 more
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invoice_Get_Ensure404WithInvalidURL_ShouldReceive404Response -->
        <test-method status="PASS" signature="invoice_Get_withInvalidUri_ShoudlGet404()[pri:0, instance:com.l1.interop.SPSPClientProxyFunctionalTest@5606c0b]" name="invoice_Get_withInvalidUri_ShoudlGet404" duration-ms="164" started-at="2016-11-18T12:54:14Z" finished-at="2016-11-18T12:54:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invoice_Get_withInvalidUri_ShoudlGet404 -->
        <test-method status="FAIL" signature="invoice_POST_ForValidInvoice_ShouldReceiveInvoiceDetailValidResponse(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.SPSPClientProxyFunctionalTest@5606c0b]" name="invoice_POST_ForValidInvoice_ShouldReceiveInvoiceDetailValidResponse" duration-ms="197" started-at="2016-11-18T12:54:14Z" data-provider="invoice_create_positive" finished-at="2016-11-18T12:54:15Z">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
The data provider is trying to pass 1 parameters but the method com.l1.interop.SPSPClientProxyFunctionalTest#invoice_POST_ForValidInvoice_ShouldReceiveInvoiceDetailValidResponse takes 3 and TestNG is unable in inject a suitable object]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
The data provider is trying to pass 1 parameters but the method com.l1.interop.SPSPClientProxyFunctionalTest#invoice_POST_ForValidInvoice_ShouldReceiveInvoiceDetailValidResponse takes 3 and TestNG is unable in inject a suitable object
	at org.testng.internal.Invoker.injectParameters(Invoker.java:1361)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1225)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
Caused by: java.lang.ArrayIndexOutOfBoundsException: 1
	at org.testng.internal.Invoker.injectParameters(Invoker.java:1359)
	... 17 more
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invoice_POST_ForValidInvoice_ShouldReceiveInvoiceDetailValidResponse -->
        <test-method status="PASS" signature="invoice_POST_ForValidInvoice_ShouldReceiveInvoiceDetailValidResponse(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.SPSPClientProxyFunctionalTest@5606c0b]" name="invoice_POST_ForValidInvoice_ShouldReceiveInvoiceDetailValidResponse" duration-ms="93" started-at="2016-11-18T12:54:14Z" data-provider="invoice_create_positive" finished-at="2016-11-18T12:54:15Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[http://localhost:8091/sample/v1/brian-123]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[ dfsp1]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[ sample_memo1]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invoice_POST_ForValidInvoice_ShouldReceiveInvoiceDetailValidResponse -->
        <test-method status="PASS" signature="invoice_POST_ForValidInvoice_ShouldReceiveInvoiceDetailValidResponse(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.SPSPClientProxyFunctionalTest@5606c0b]" name="invoice_POST_ForValidInvoice_ShouldReceiveInvoiceDetailValidResponse" duration-ms="50" started-at="2016-11-18T12:54:15Z" data-provider="invoice_create_positive" finished-at="2016-11-18T12:54:15Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[http://localhost:8091/sample/v1/sam-123]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[ dfsp2]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[ sample_memo2]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invoice_POST_ForValidInvoice_ShouldReceiveInvoiceDetailValidResponse -->
        <test-method status="PASS" signature="invoice_POST_ForValidInvoice_ShouldReceiveInvoiceDetailValidResponse(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.SPSPClientProxyFunctionalTest@5606c0b]" name="invoice_POST_ForValidInvoice_ShouldReceiveInvoiceDetailValidResponse" duration-ms="47" started-at="2016-11-18T12:54:15Z" data-provider="invoice_create_positive" finished-at="2016-11-18T12:54:15Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[http://localhost:8091/sample/v1/murthy-123]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[ dfsp3]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[ sample_memo3]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invoice_POST_ForValidInvoice_ShouldReceiveInvoiceDetailValidResponse -->
        <test-method status="PASS" signature="invoice_POST_testingFor400BadRequest()[pri:0, instance:com.l1.interop.SPSPClientProxyFunctionalTest@5606c0b]" name="invoice_POST_testingFor400BadRequest" duration-ms="49" started-at="2016-11-18T12:54:15Z" finished-at="2016-11-18T12:54:15Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invoice_POST_testingFor400BadRequest -->
        <test-method status="PASS" signature="invoice_POST_testingFor404ResourceNotFound()[pri:0, instance:com.l1.interop.SPSPClientProxyFunctionalTest@5606c0b]" name="invoice_POST_testingFor404ResourceNotFound" duration-ms="30" started-at="2016-11-18T12:54:15Z" finished-at="2016-11-18T12:54:15Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invoice_POST_testingFor404ResourceNotFound -->
        <test-method status="PASS" signature="invoice_POST_testingFor415UnsupportedMediaType()[pri:0, instance:com.l1.interop.SPSPClientProxyFunctionalTest@5606c0b]" name="invoice_POST_testingFor415UnsupportedMediaType" duration-ms="74" started-at="2016-11-18T12:54:15Z" finished-at="2016-11-18T12:54:15Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invoice_POST_testingFor415UnsupportedMediaType -->
        <test-method status="PASS" signature="afterClass()[pri:0, instance:com.l1.interop.SPSPClientProxyFunctionalTest@5606c0b]" name="afterClass" is-config="true" duration-ms="0" started-at="2016-11-18T12:54:15Z" finished-at="2016-11-18T12:54:15Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
      </class> <!-- com.l1.interop.SPSPClientProxyFunctionalTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
